example/src/main/tut/typeclass/Apply.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: docs
+title:  "Apply"
+---
+
+# Apply
+
+*A strong lax semi-monoidal endofunctor. This is equivalent to an Applicative without pure.*
+
+In addition to the laws of a [Functor](./Functor.html), an Apply instance needs to satisfy the following law:
+
+- Associativity: `x.ap(y).ap(z) === x.ap(y.ap(z)).map { case (x, (y, z)) => ((x, y), z) }`
```

Comment:
Is this the law? It looks like you're talking in terms of `product` or `tuple2` or other such `Apply`-based functions.

Reply:
Oh, did I mess this up? I'll look into it.